ptex.pc.in

```diff
@@ -0,0 +1,21 @@
+# this template is filled-in by CMake `configure_file(... @ONLY)`
+# the `@....@` are filled in by CMake configure_file(),
+# from variables set in your CMakeLists.txt or by CMake itself
+#
+# Good tutoral for understanding .pc files:
+# https://people.freedesktop.org/~dbn/pkg-config-guide.html
+
+prefix="@CMAKE_INSTALL_PREFIX@"
+exec_prefix="${prefix}"
+libdir="${prefix}/lib"
```

Comment:
A slightly better value for libdir would be `${prefix}/@CMAKE_INSTALL_LIBDIR@` so that we can have a custom lib directory and it'll work as expected.

The pattern above only handles the scenario where CMAKE_INSTALL_LIBDIR is a prefix-relative path.

It doesn't handle the case where CMAKE_INSTALL_LIBDIR is an absolute path. That can be handled by calculating a variable in cmake so that we can use it here.

https://github.com/wdas/SeExpr/commit/b2656fbacf59eb8e2bcee8aefb39d40eb76ec94b#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR272

is an example where a CMAKE_INSTALL_PKGCONFIG_LIBDIR variable is calculated to handle that scenario and then the .pc.in file can contain `libdir="@CMAKE_INSTALL_PKGCONFIG_LIBDIR@"`

Reply:
Ditto for includedir and then I think this is just about ready to merge. Thanks for adding pkg-config support

Reply:
Last sug ~ Please move these new files to `src/build/` and cmake updates to `src/build/CMakeLists.txt` alongside the existing cmake packaging and squash those tweaks into the original commits. That way all of the build/packaging-related stuff is under `src/build/`.

Reply:
I addressed your remarks and changed the resulting ptex.pc file to lowercase as all the files in /usr/share/pkgconfig on my Ubuntu machine seem to be lowercase.